### ISSUE TITLE
wrong _currentIndex calculation for _minDate.timeOfDay > _currentDate.timeOfDay  // DayView

### ISSUE THREAD

**Issue by NicolaiRahm** [SimformSolutionsPvtLtd/flutter_calendar_view#80]
I have encountered an issue with the _currentIndex calculation for the DayView.
To not have the events displayed one day off when the _minDate time of the day is greater than that of the _currentDate
the _currentDate and _minDate should be "normalized" to the same hour e.g.
```
DateTime _currentNormalizedToHour = DateTime(_currentDate.year, _currentDate.month, _currentDate.day, 0);
DateTime _minNormalizedToHour = DateTime(_minDate.year, _minDate.month, _minDate.day, 0);
_currentIndex = _currentNormalizedToHour.getDayDifference(_minNormalizedToHour);
// _currentIndex = _currentDate.getDayDifference(_minDate);
```

To also always get the expected _totalDays _maxDate should also be normalized.

It would therefore probably be most convenient to change the `getDayDifference` method

**Comment by francescovgg** [comment]
I guess the wrong _currentIdex calculation is due to DateTime.difference and Duration.inDays() disalignment when considering Daylight Saving Time, as also stated in DateTime.difference() documentation.

To fix this we have to compare dates in utc, e.g.:

```
/// Gets difference of days between [date] and calling object.
int getDayDifference(DateTime date) => 
    DateTime.utc(year, month, day).difference(DateTime.utc(date.year, date.month, date.day)).inDays.abs();
```

**Comment by faiyaz-shaikh** [comment]
@NicolaiRahm Can you please check on latest release.

**Comment by faiyaz-shaikh** [comment]
Fixed in 1.0.1 PR #136
